CITi-UFPE/hackapride_2022
# Feature/awards

# What I did?

- Structure the content of the container

## How to test?

1. Open the terminal and go to project folder

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout feature/Awards`
   1. `git pull origin feature/Awards`
   1. `yarn install` ou `npm install`
   1. `code .`
   1. `yarn start` ou `npm start`

1. In browser go to http://localhost:3000/

1. Verify if the content is equal to the mockup
